src/rhtap/postcreation/strategies/commands/triggerJenkinsJobCommand.ts

```
@@ -0,0 +1,20 @@
+import { JenkinsCI } from "../../../core/integration/ci";
+import { BaseCommand } from "./baseCommand";
+
+export class TriggerJenkinsJobCommand extends BaseCommand {
+
+    public async execute(): Promise<void> {
+        this.logStart('Triggering Jenkins job');
+
+        const jenkinsCI = this.ci as JenkinsCI;
```

ESLint

[error] Replace `········` with `····`

(prettier/prettier)